flight/targets/brainre1/board-info/board_hw_defs.c

```diff
@@ -68,7 +71,12 @@ static const struct pios_annunc pios_annuncs[] = {
 		},
 		.remap = 0,
 		.active_high = true,
-	}
+	},
+#if defined(PIOS_INCLUDE_RE1_FPGA)
```

Comment:
Since this is the BrainRE1 target, and the FPGA is an integral part of it, are lines 75 and 79 necessary?